### ISSUE TITLE
RM_DUP unset  in config file has no effect

### ISSUE THREAD

**Issue by gundalav** [nservant/HiC-Pro#256]
Hi Nicholas, 

I have unset the parameter in the config:

from 

`RM_DUP = 1 `

into 

`RM_DUP =`

However the statistics as shown in `allValidPairs.mergestat` file still show the difference
between `valid_interaction_rmdup` and `valid_interaction`. I expect the two values should be the same with `RM_DUP` unset.

```
valid_interaction	56356237
valid_interaction_rmdup	13301277
trans_interaction	675429
cis_interaction	12625848
cis_shortRange	11451768
cis_longRange	1174080
```

Please correct me if I'm wrong. If so, what is the indicator that `RM_DUP =` is really working?

Thanks and hope to hear from you again.

G.V

**Comment by nservant** [comment]
Hi,
Which version of HiC-Pro are you using please ?
Could you try with RM_DUP = 0 ?
Thanks

**Comment by gundalav** [comment]
Hi Nicholas, 

> Which version of HiC-Pro are you using please ?

HiC-Pro_2.11.1

G.V.

**Comment by gundalav** [comment]
Thanks.

> Could you try with RM_DUP = 0 ?

This works. I thought it may be good to make it more consistent with `MIN_CIS_DIST`.
It accepts `MIN_CIS_DIST =` instead of `MIN_CIS_DIST = 0`.

G.V.

**Comment by nservant** [comment]
ok. Will check that later on
N

**Comment by nservant** [comment]
Thanks. Fixed in 2.11.3
